src/core/annotation.js

```diff
@@ -2780,6 +2781,25 @@ class TextWidgetAnnotation extends WidgetAnnotation {
       !this.hasFieldFlag(AnnotationFieldFlag.FILESELECT) &&
       this.data.maxLen !== 0;
     this.data.doNotScroll = this.hasFieldFlag(AnnotationFieldFlag.DONOTSCROLL);
+
+    // Check if we have a date or time.
+    const {
+      data: { actions },
+    } = this;
+    if (actions?.Keystroke?.[0]) {
```

Comment:
Could there be multiple actions defined here, and if so, should we perhaps loop over them and break if we find a match instead of always picking the first one?